### PR TITLE
feat: Add comment `Fixes #issue` when merging PRs

### PR DESCRIPTION
**Pull request by zregvart**
This adds a comment `Fixes #issue` to the pull request after it's been merged.

I have no idea how to test this, but it compiles so that's something.

@jimmidyson does this make any sense?

### CODE REVIEW

File: pkg/webhook/merge_on_review_approval_and_status_checks.go [hunk 1]
```diff
@@ -169,5 +169,13 @@ func mergePR(issue *github.Issue, owner, repository string, gh *github.Client, c
 		return errors.Wrapf(err, "failed to merge pull request %s", issue.GetHTMLURL())
 	}
 
+	comment := fmt.Sprintf("Fixes #%d", issue.GetNumber())
```
Comment (jimmidyson) [inline pkg/webhook/merge_on_review_approval_and_status_checks.go]: With the github API, a PR is also an issue... so `issue.GetNumber()` is actually the PR number... so this comment is saying the PR fixes itself!

Comment (zregvart) [reply pkg/webhook/merge_on_review_approval_and_status_checks.go]: doh, let me have another go, see what I did there :)

**Comment by zregvart** [comment]
@jimmidyson perhaps this might work

**Comment by zregvart** [comment]
If we follow the eng. guidelines then there should be a `Fixes #xxx` in PR and commit message, that would make this superfluous.

**Pull request closed without merging**